### DISCUSSION TITLE
16

### DISCUSSION THREAD

**Discussion by giscus** [car-ffeine/car-ffeine.github.io#11] [Announcements]
# 16

안녕하세요 박스터 입니다.

https://car-ffeine.github.io/16

**Comment by BackFoxx** [comment]
ID를 서버에서 미리 만들어서 저장하면 성능상 더 좋은 줄 알았는데 사실 별 차이가 없어지는 거였군요 👀
펏시스턴스를 오버라이딩해서 isNew 조건을 ID여부에서 createdAt 여부로 바꾼 것도 놀랍습니다 멋져요 👍🏻

**Comment by be-student** [comment]
진짜 많이 배웠네요
isNew 조건이 기본으로 저렇게 되어있었군요

**Comment by greeng00se** [comment]
좋은 글 감사합니다 :hurricane:

**Comment by Choi-JJunho** [comment]
좋은글 감사합니다 :박스터펫:

**Comment by hgo641** [comment]
좋은글 감사합니다 :박스터슝슝:

**Comment by sosow0212** [comment]
오우 깊게 파보셨네요 덕분에 새로운 것들을 많이 배워갑니다 👍

**Comment by green-kong** [comment]
와 타이칸 받을만 하네요;; 멋져요
